test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -458,6 +458,7 @@ public async Task GivenListOfResources_WhenSearchedWithCountSummary_ThenTotalCou
         [InlineData("_summary", "_count")]
         [InlineData("_summary", "xyz")]
         [InlineData("_count", "abc")]
+        [InlineData("_elements", "")]
         [Theory]
         [Trait(Traits.Priority, Priority.One)]
         public async Task GivenResources_WhenSearchedWithIncorrectSummaryParams_ThenExceptionShouldBeThrown(string key, string val)
```

Comment:
Can the name of this test be updated? It is testing more than just the  _summary param.